client/src/components/navigation/Navbar.js

```diff
@@ -17,28 +18,25 @@ const useStyles = makeStyles({
 });
 
 // TODO: Add functionality for page changes when react-router is implemented
-const Navbar = () => {
+const Navbar = (props) => {
```

Comment:
Just curious is NavBar being used anywhere? I noticed we removed it in App.js

Reply:
It's on line 26 in App.js!

Reply:
ah lol Thanks!